### PR TITLE
D4T-807 Remove scope d4tv1 from token requests

### PR DESCRIPTION
**Pull request by bozimmerman**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ runs:
       shell: bash
       run: |
         # Generate console credential file from env vars
-        echo "{\"consoleCredentials\":{\"audience\":\"https://axis.snxd.com/\",\"clientId\":\"${{ inputs.solsta_client_id }}\",\"clientSecret\":\"${{ inputs.solsta_client_secret }}\",\"grant\":\"clientCredentials\",\"scope\":\"d4tv1\"}}"  > client_credentials.json
+        echo "{\"consoleCredentials\":{\"audience\":\"https://axis.snxd.com/\",\"clientId\":\"${{ inputs.solsta_client_id }}\",\"clientSecret\":\"${{ inputs.solsta_client_secret }}\",\"grant\":\"clientCredentials\",\"scope\":\"\"}}"  > client_credentials.json
```
Comment (nmoinvaz) [inline action.yml]: Strange that scope is empty string rather than removed. Perhaps @steve-tucker will have input on it.

Comment (steve-tucker) [reply action.yml]: I think it should probably be removed.

Comment (bozimmerman) [reply action.yml]: I will do so right now.  
Also, a Question: It would work the same either way?  IOW, is this change cosmetic or functional?

Comment (steve-tucker) [reply action.yml]: tbh mostly undefined behavior, but there are tinges of cosmetic. I know what the behavior for a request with a missing scope is, I know what it is for a filled in scope. But I'm not sure what the oauth behavior for scope being an empty string on the login request. If you look it up and tell me it's defined in the specification, then I'll be satisfied also.

**Review by steve-tucker — APPROVED** [review]

**Pull request merged**